examples/pytorch/image-pretraining/run_mim.py

```diff
@@ -359,7 +363,10 @@ def main():
         IMAGE_PROCESSOR_TYPES = {
             conf.model_type: image_processor_class for conf, image_processor_class in IMAGE_PROCESSOR_MAPPING.items()
         }
-        image_processor = IMAGE_PROCESSOR_TYPES[model_args.model_type]()
+        if model_args.fast_image_processor:
+            image_processor = IMAGE_PROCESSOR_TYPES[model_args.model_type][1]()
+        else:
+            image_processor = IMAGE_PROCESSOR_TYPES[model_args.model_type][0]()
```

Comment:
Let's use fast by default?

```suggestion
        image_processor = IMAGE_PROCESSOR_TYPES[model_args.model_type][-1]()
```